Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -7119,6 +7118,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEFINES_MODULE = YES;
+				DEVELOPMENT_TEAM = 48YBP49Y5N;
```

Comment:
Do you know what caused this change? Should the development team be under the group above?

Reply:
I don't know the cause of the change.